Tech Lead/Minutes/20250304.md

```diff
@@ -0,0 +1,53 @@
+    - Julien Lenhardt [@jlenh](https://github.com/jlenh)
+
+## Action Items (from 4 February: all done!)
+
+| Meeting | Assignees | Item | Status |
+|-|-|-|-|
+| 2024-12-03 | @rbeucher | Arrange meeting on Intake | happened |
+| 2024-12-03 | ~~@ehogan~~ @bouweandela | Split docs meeting | happened, @flicj191 will start working on this, see https://github.com/esmValGroup/esMValTool/issues/3860 for further information |
+| 2025-02-04 | @valeriupredoi | set up TLT-SLT call for broken recipes (see below) | meeting happened |
+
+## Agenda and topics discussed
+
+- Pending actions: both have been addressed: docs split will be started by @flicj91 (cheers!) and meeting about the Intake lib has happened, also the SLT-TLT meeting on broken recipes has happened, see Discsssion item and linked PR by Axel: https://github.com/ESMValGroup/ESMValTool/discussions/3912
+
+## Agenda
+
+- Status of release
+    - Core has been released!
+    - Tool stable release out today: one test not picking up the Core 2.12, and remember to pin iPython
+- @ehogan: Sharing from the User Engagement Team (UET): Two tutorial-related items requiring feedback from the TLT:
+  - Proposal to divide the tutorial into two repositories. This has the benefit of enabling adding other advanced tutorials, e.g. from ACCESS-NRI & MO, domain-specific, etc., without requiring coordination / merging with the main tutorial(s) below:
+    - a basic tutorial that gets updated regularly (and tagged to match the ESMValTool version)
+    - an advanced tutorial that is more community driven and may not be updated as regularly
+    - @boueandela: this has always been the aim - so it's all cool
+  - Rather than using the new Software Carpentry template (I am struggling to see the benefits of continuing to use the Software Carpentry template), can we consider using what we are currently using for the documentation? Benefits include:
+    - there is no need to maintain difficult-to-build R-based environments
+    - the tutorial(s) could be hosted via Read the Docs
+    - making changes to the tutorial(s) should be easier, due to familiarity of the structure / syntax (it would be the same as the documentation)
+    - the new Software Carpentry theme (example: https://swcarpentry.github.io/git-novice/) does not look dissimilar to the (Sphinx + PyData) theme
+  - discussion on this: we like, but TLT can't really spare time to work, apart from helping with configuration and setup
+  - We could provide a template repository within ESMValGroup that people could use as a starting point for their own tutorial(s).
+  - @bouweandela and @charles-turner-1: happy with Intake? It's a lot of technical work with packaging and internal `$PATH` issues, conda env issues etc
+  - @jlenh: ESMValTool `main` branch used for the REF, adding dependencies: @bouwenadela careful with license, and then be maintained
+  - @bettina-gier: tests for CMORizers? That a good idea - pls open a GH issue or Discussion item
```

Comment:
Feel free to link: https://github.com/ESMValGroup/ESMValTool/issues/3946

Reply:
good call, done in 9c9d3e5bb351b6939af0d45cee3f6718180d7e0f